sql/ha_partition.h

```diff
@@ -1303,18 +1304,7 @@ class ha_partition final :public handler
       The following code is not safe if you are using different
       storage engines or different index types per partition.
     */
-    ulong part_flags= m_file[0]->index_flags(inx, part, all_parts);
-
-    /*
-      The underlying storage engine might support Rowid Filtering. But
-      ha_partition does not forward the needed SE API calls, so the feature
-      will not be used.
-
-      Note: It's the same with IndexConditionPushdown, except for its variant
-      of IndexConditionPushdown+BatchedKeyAccess (that one works). Because of
-      that, we do not clear HA_DO_INDEX_COND_PUSHDOWN here.
-    */
-    return part_flags & ~HA_DO_RANGE_FILTER_PUSHDOWN;
+    return m_file[0]->index_flags(inx, part, all_parts);
```

Comment:
Do I interpret it correctly that this will make ha_partition also support Rowid Filtering? 

This might be good, but then the commit comment needs to mention this and there should be test coverage.

I think it's better to enable Rowid Filtering in a separate patch.  Unless the effort to enable ICP and Rowid Filtering at the same time is much less than enabling them one-by-one.